lib/kamal/cli/main.rb

```diff
@@ -6,6 +6,8 @@ def setup
         say "Ensure Docker is installed...", :magenta
         invoke "kamal:cli:server:bootstrap"
 
+        run_hook "docker-setup"
```

Comment:
Do we want to always run this, or only run it when docker is actually installed?

Either we should move calling the hook inside `kamal:cli:server:bootstrap` so that it is also called when that command is invoked directly.

Reply:
Yes, good shout, that does make more sense :+1: